.github/workflows/temp-branch-build-and-push.yaml

```diff
@@ -0,0 +1,47 @@
+name: Branch - Build and push docker image
+
+on:
+  push:
+    branches:
+      - "ertugrul/change-to-binary"
```

Comment:
I'll keep this file version controlled actually so that we avoid copying it over for stage manual tests

Reply:
Can we not just make this not based on a branch on based on anything but main?

Reply:
This will then push a lot of images on each commit and most of them won't be used for stage testing. Our image size is pretty big 😅